scenario/uploaddata.go

```diff
@@ -70,124 +75,103 @@ func (settings UploadDataSettings) Execute(
 		return
 	}
 
-	restHandler := sessionState.Rest
+	sessionState.Rest.GetAsync(
+		fmt.Sprintf("%s/%s/quota", host, datafileEndpoint), actionState, sessionState.LogEntry, nil,
+	)
 
 	dataConnectionID, err := sessionState.FetchDataConnectionID(actionState, host, settings.SpaceID)
```

Comment:
`FetchDataConnectionID `needs to be updated to /api/v1/dc-dataconnections/DataFiles?type=connectionname